src/btree/bt_read.c

```diff
@@ -68,6 +68,129 @@ __row_instantiate(WT_SESSION_IMPL *session,
 	return (0);
 }
 
+/*
+ * __wt_find_lookaside_upd --
+ * 	Scan the lookaside for a record the btree cursor wants to position on.
+ * 	Create an update for the record and return to the caller.
+ */
+int
+__wt_find_lookaside_upd(WT_SESSION_IMPL *session,
+    WT_CURSOR_BTREE *cbt, WT_UPDATE **updp, bool ignore_birthmark)
+{
+	WT_CACHE *cache;
+	WT_CURSOR *cursor;
+	WT_DECL_RET;
+	WT_ITEM *key;
+	WT_ITEM las_key, las_value;
+	WT_REF *ref;
+	WT_UPDATE *upd;
+	wt_timestamp_t durable_timestamp, las_timestamp;
+	size_t incr;
+	uint64_t las_counter, las_pageid, las_txnid;
+	uint32_t las_id, session_flags;
+	uint8_t prepare_state, upd_type;
+	int cmp;
+	bool locked, upd_visible;
+
+	cache = S2C(session)->cache;
```

Comment:
It helps people figure out if there's any missing initializations if local variable initialization is roughly in the same order as the declarations. Not a rule, but a guideline. :)